index.bs

```diff
@@ -1598,286 +1600,161 @@ A URL for Blob and File reference</h2>
+  (that is, the Unicode [=code point=] sequence U+0062, U+006C, U+006F, U+0062)
   to |result|.
-1. Append the ":" (U+003A COLON) character to |result|.
+1. Append U+003A COLON (:) to |result|.
 1. Let |settings| be the [=current settings object=]
 1. Let |origin| be |settings|'s [=environment settings object/origin=].
 1. Let |serialized| be the <a lt="ASCII serialization of an origin">ASCII serialization</a> of |origin|.
 1. If |serialized| is "null", set it to an implementation-defined value.
 1. Append |serialized| to |result|.
-1. Append the "/" character (U+0024 SOLIDUS) to |result|.
-1. Generate a UUID [[RFC4122]] as a Unicode string and append it to |result|.
+1. Append U+0024 SOLIDUS (/) to |result|.
+1. Generate a UUID [[!RFC4122]] as a Unicode string and append it to |result|.
 1. Return |result|.
 
-<h4 id="fragmentDiscussion">
-Discussion of Fragment Identifier</h4>
-
-The <a for=url>fragment</a>'s resolution and processing directives depend on the media type [[RFC2046]]
-of a potentially retrieved representation,
-even though such a retrieval is only performed if the <a>Blob URL</a> is <a>dereferenced</a>.
-For example, in an HTML file [[HTML]] the fragment could be used to refer to an anchor within the file.
-If the user agent does not recognize the media type of the resource,
-OR if a fragment is not meaningful within the resource,
-it must ignore the fragment.
-The fragment must not be used to identify a resource;
-only the <code>blob</code> [=url/scheme=] and the [=url/origin=]/[=url/path=] of the <a>URL</a> constitute a valid resource identifier.
-
-A valid Blob URL reference could look like:
-<code>blob:http://example.org:8080/550e8400-e29b-41d4-a716-446655440000#aboutABBA</code>
-where "#aboutABBA" might be an HTML fragment identifier referring to an element with an id attribute of "aboutABBA".
-
-Note: The <a for=url>fragment</a> is not used to identify the resource.
-The <code>URL.{{URL/createObjectURL()}}</code> method does not generate a fragment.
-
+</div>
 
-<h3 id="requestResponseModel">
+<h4 id="requestResponseModel">
 Dereferencing Model for Blob URLs</h3>
 
-*This section is informative.*
-
-Note: The [[!URL]] and [[!Fetch]] specifications should be considered normative for parsing and fetching <a>Blob URLs</a>.
+<div algorithm="resolveURL">
+To <dfn for="Blob URL" lt="resolve">resolve a Blob URL</dfn> given a |url| (a [=URL=]), run the following steps:
 
-<a>Blob URLs</a> are <dfn lt="dereference">dereferenced</dfn>
-when the user agent retrieves the resource identified by the Blob URL
-and returns it to the requesting entity.
-This section provides guidance on [=/requests=] and [=/responses=]
+1. [=Assert=]: |url|'s [=url/scheme=] is <code>"blob"</code>.
+1. Let |store| be the user agent's [=Blob URL Store=];
+1. Let |url string| be the result of [=URL serializer|serializing=] |url| with the *exclude fragment flag* set.
+1. Let |entry| be the result of [=map/getting=] |store|[|url string|].
```

Comment:
You can just say "Let |entry| be |store|[|url string|]." Or even "Return |store|[|url string|]."